app/boards/shields/waterfowl/Kconfig.defconfig

```diff
@@ -0,0 +1,53 @@
+
+if SHIELD_WATERFOWL_LEFT
+
+config ZMK_KEYBOARD_NAME
+	default "Waterfowl"
+
+config ZMK_SPLIT_BLE_ROLE_CENTRAL
+	default y
+
+endif
+
+if SHIELD_WATERFOWL_LEFT || SHIELD_WATERFOWL_RIGHT
+
+config ZMK_SPLIT
+	default y
+ 
+if ZMK_DISPLAY
+
+config I2C
+	default y
+
+config SSD1306
+	default y
+
+config SSD1306_REVERSE_MODE
+	default y
+
+endif # ZMK_DISPLAY
+
+if LVGL
+
+config LVGL_HOR_RES_MAX
+	default 128
+
+config LVGL_VER_RES_MAX
+	default 64
+
```

Comment:
The two resolution Kconfigs are no longer necessary since https://github.com/zmkfirmware/zmk/commit/da209c453eac57b9bb3818a81ad9b20345618c4b